great_expectations/data_context/store/_store_backend.py

```diff
@@ -46,7 +46,11 @@ def __init__(
         """  # noqa: E501
         self._fixed_length_key = fixed_length_key
         self._suppress_store_backend_id = suppress_store_backend_id
-        self._manually_initialize_store_backend_id = manually_initialize_store_backend_id
+        self._manually_initialize_store_backend_id: str = (
```

Comment:
This ensures `self._manually_initialize_store_backend_id` as a `str` which is what this class previously assumed this to be.